apps/site/src/components/PageLanding/CommunityCard.tsx

```diff
@@ -15,7 +21,7 @@ export default function CommunityCard({ name, image, url }: Props) {
         height={476}
         alt={name}
         placeholder="empty"
```

Comment:
```suggestion
        placeholder="blur"
```

Set to `blur` instead. Then make the images statically imported so that next.js can automatically populate the `blurDataURL`. (`communities.json` will have to be changed to `communities.ts`, and the `CommunityCard`s `image` prop type should be set to `StaticImageData`).